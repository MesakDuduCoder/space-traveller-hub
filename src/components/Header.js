import React from 'react';

const Header = () => (
  <header>
    <nav />
  </header>
);

export default Header;
