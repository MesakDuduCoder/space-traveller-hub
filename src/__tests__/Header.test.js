import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Header from '../components/Header';

it('renders correctly', () => {
  const header = renderer
    .create(
      <MemoryRouter>
        <Header />
      </MemoryRouter>,
    )
    .toJSON();
  expect(header).toMatchSnapshot();
});
