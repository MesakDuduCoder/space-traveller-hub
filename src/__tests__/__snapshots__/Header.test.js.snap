// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <header>
    <nav>
      <div
        className="logo"
      >
        <img
          alt="planet"
          src="planet.png"
        />
        <span
          className="title"
        >
          Space Travelers' Hub
        </span>
      </div>
      <div
        className="links"
      >
        <a
          activeclassname="active"
          aria-current="page"
          className="active"
          href="/"
          onClick={[Function]}
        >
          Rockets
        </a>
        <a
          activeclassname="active"
          className=""
          href="/missions"
          onClick={[Function]}
        >
          Missions
        </a>
        <span
          className="line"
        />
        <a
          activeclassname="active"
          className=""
          href="/profile"
          onClick={[Function]}
        >
          My Profile
        </a>
      </div>
    </nav>
  </header>,
  <hr />,
]
`;
